src/widgets/TOCWidget.vue

```diff
@@ -22,13 +22,30 @@
       };
     },
     computed: {
+      endpoint() {
+        return 'https://sv-mini-atlas.herokuapp.com';
+      },
+      rootToc() {
+        return `${this.endpoint}/tocs/demo-root.json`;
+      },
       url() {
-        return 'https://sv-mini-atlas.herokuapp.com/tocs/toc.oaf-1.json';
+        // TODO: Temporary (hack) logic until the TOC API hardens.
```

Comment:
I think from when we wrote up the story to when we generated the TOCs, we weren't sure about id.  I think it'd be ok to use `@id` rather than having to create another layer (even at this early stage)

Reply:
Have just pivoted to URN and it is much better all round.